test/dynamics/models/test_rotating_frame.py

```diff
@@ -651,19 +686,21 @@ def test_instantiation_errors(self):
     def test_jitting(self):
         """Test jitting of state_into_frame and _conjugate_and_add."""
 
-        rotating_frame = RotatingFrame(Array([1.0, -1.0]))
+        from jax import jit
+
+        rotating_frame = RotatingFrame(self.asarray([1.0, -1.0]))
 
-        self.jit_wrap(rotating_frame.state_into_frame)(t=0.1, y=np.array([0.0, 1.0]))
-        self.jit_wrap(rotating_frame._conjugate_and_add)(
-            t=0.1, operator=np.array([[0.0, 1.0], [1.0, 0.0]])
+        jit(rotating_frame.state_into_frame)(t=0.1, y=self.asarray([0.0, 1.0]))
+        jit(rotating_frame._conjugate_and_add)(
+            t=0.1, operator=self.asarray([[0.0, 1.0], [1.0, 0.0]])
         )
 
     def test_jit_and_grad(self):
         """Test jitting and gradding of state_into_frame and _conjugate_and_add."""
 
-        rotating_frame = RotatingFrame(Array([1.0, -1.0]))
+        from jax import jit, grad
 
-        self.jit_grad_wrap(rotating_frame.state_into_frame)(0.1, np.array([0.0, 1.0]))
-        self.jit_grad_wrap(rotating_frame._conjugate_and_add)(
-            0.1, np.array([[0.0, 1.0], [1.0, 0.0]])
-        )
+        rotating_frame = RotatingFrame(self.asarray([1.0, -1.0]))
+
+        jit(grad(rotating_frame.state_into_frame))(0.1, self.asarray([0.0, 1.0]))
+        jit(grad(rotating_frame._conjugate_and_add))(0.1, self.asarray([[0.0, 1.0], [1.0, 0.0]]))
```

Comment:
Can we drop it? Or need we to solve this by defining a helper function that sums the return up to calculate `grad`?